### DISCUSSION TITLE
Where is a changelog?

### DISCUSSION THREAD

**Discussion by zebapy** [stoplightio/elements#2521] [General]
I can't find a changelog anywhere obvious (like CHANGELOG.md in root repo) nor do releases/tags seem to have info. 

What was the major breaking change in v7 -> v8?

**Comment by marcelstoer** [comment]
I am evaluating Elements and was looking for a changelog or release notes as well. Would one of the current maintainers (e.g. @ben-smartbear) care to point us into the right direction, please.

The team seems to have stopped creating tags in the Git repo; the last ones are very old. npmjs [has all the versions](https://www.npmjs.com/package/@stoplight/elements?activeTab=versions) but no documentation about them of course. The documentation doesn't contain any release notes either.
